extension/options.html

```diff
@@ -120,6 +120,56 @@
 
     </section>
 
+    <section class="panel">
+        <h2>Site profiles</h2>
+        <table id="saved_site_profiles">
+            <thead>
+                <tr>
+                    <th>Domain</th>
+                    <th>Change domain to</th>
+                    <th>Append string</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody></tbody>
+        </table>
+    </section>
+
+    <section class="panel">
+        <form id="newSiteProfile">
+            <h2>Create new site profile</h2>
+
```

Comment:
As you did a great job with the existing documentation and design, I think it is best when you add this explanation the way you think is best.